### PR TITLE
IDVA5-157-sole-trader-confirm-correspondence-address

### PR DESCRIPTION
**Pull request by abhiahirv1**
Added Confirm your correspondence address Screen

### CODE REVIEW

File: src/main/config/index.ts [hunk 1]
```diff
@@ -8,3 +8,4 @@ export const SOLE_TRADER_KICK_OUT = `${BASE_SOLE_TRADER_URL}/kick-out/stop-not-r
 export const SOLE_TRADER_SECTOR_YOU_WORK_IN = `${BASE_SOLE_TRADER_URL}/sector-you-work-in/sector-you-work-in`;
 export const SOLE_TRADER_WHERE_DO_YOU_LIVE = `${BASE_SOLE_TRADER_URL}/where-do-you-live/where-do-you-live`;
 export const SOLE_TRADER_MANUAL_CORRESPONDANCE_ADDRESS = `${BASE_SOLE_TRADER_URL}/correspondance-address-manual/capture-correspondance-address-manual`;
+export const SOLE_TRADER_CONFIRM_CORRESPONDANCE_ADDRESS = `${BASE_SOLE_TRADER_URL}/correspondance-address-confirm/correspondance-address-confirm`;
```
Comment (swalke-ch) [inline src/main/config/index.ts:11]: URLs should defined in pageURL.ts page same should be used in controller and other places.

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  <main class="govuk-main-wrapper" id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <form class="form" action="address-correspondence-confirm" method="post">
+
+          <span class="govuk-caption-l">{{firstName}} {{lastName}}</span>
+
+          <h1 class="govuk-heading-l">Confirm the correspondence address</h1>
```
Comment (swalke-ch) [inline src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk:12]: MIssiing localisation

**Review by swalke-ch — APPROVED** [review]

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+              <li>{{correspondenceAddress.line1}}</li>
+              <li>{{correspondenceAddress.line2}}</li>
+              <li>{{correspondenceAddress.town}}</li>
+              <li>{{correspondenceAddress.county}}</li>
+              <li>{{correspondenceAddress.country}}</li>
+              <li>{{correspondenceAddress.postcode}}</li>
+          </ul>
+
+          <p class="govuk-body">
+            <a href="correspondance-address-manual" class="govuk-link govuk-link--no-visited-state">Edit address</a>
```
Comment (swalke-ch) [inline src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk:24]: can this href be pulled from the pageURL ?

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+          </p>
+          <button class="govuk-button" data-module="govuk-button" onclick="confirmAndContinue()">Confirm and continue</button>
```
Comment (swalke-ch) [inline src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk:26]: why do we need onclick event ?

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/config/index.ts (hunk 1 above)
Comment (abhiahirv1) [reply src/main/config/index.ts]: Done that

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk (hunk 2 above)
Comment (abhiahirv1) [reply src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk]: Added

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk (hunk 4 above)
Comment (abhiahirv1) [reply src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk]: Removed that.

File: src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk (hunk 3 above)
Comment (swalke-ch) [reply src/main/views/features/sole-trader/correspondance-address-confirm/correspondance-address-confirm.njk]: PageURLs is not about localisation. what will happens if you add this in pageURLs?

**Review by sardhani-ch — DISMISSED** [review]
LGTM

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by sardhani-ch — APPROVED** [review]

**Pull request merged**